docs/migration.md

```diff
@@ -2473,7 +2474,8 @@ See [this document](comparison.md#host-metrics) to learn more.__
 
 ### Local Windows Event Log Source (LocalWindowsEventLog)
 
-Local Windows Event Log Source is not supported by the OpenTelemetry Collector.
+There is no migration process from Installed Collector to OpenTelemetry Collector.
+In order to use OpenTelemetry Collector, dedicated app needs to be installed.
```

Comment:
What do you mean here by `dedicated app`?

Reply:
I don't think there is any documentation yet I can relate to.

Reply:
```suggestion
In order to use OpenTelemetry Collector, dedicated Sumo Logic app needs to be
installed.
```

Reply:
@perk-sumo yes, that's what I meant